app/models/movie.rb

```diff
@@ -1,3 +1,14 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :actor_movies
+  has_many :actors, through: :actor_movies
+
+  def average_age(actors)
+    age = 0
+    actors.each do |actor|
+      age += actor.age
+    end
+    age = age / actors.count
+    age
+  end
```

Comment:
We'd also like to see a method here (instead of controller) to order the actors:
```
def order_youngest_first
    actors.order(age: :asc)
end
```

Reply:
then, in the view, you can call this on a movie instance and iterate over the actors: `@movie.order_actors_ youngest_first.each do |actor|`